src/common/tests_dmg_helpers.c

```diff
@@ -487,6 +488,19 @@ dmg_pool_create(const char *dmg_config_file,
 					    entry->dpe_str);
 			if (args == NULL)
 				D_GOTO(out, rc = -DER_NOMEM);
+		} else {
+			static bool need_init_rand = true;
+
+			if (need_init_rand) {
+				srand((unsigned int)time(NULL));
+				need_init_rand = false;
+			}
+
+			/* pool label is required, generate one randomly */
+			args = cmd_push_arg(args, &argcount, "--label=test-%x ",
```

Comment:
`-` is not a valid label character, so this is going to fail. I'm not sure I agree with that decision, but it was made to avoid allowing uuids to supplied as labels.

Reply:
Whoops. Fixed.